### DISCUSSION TITLE
How to change/remap/namespace the topic to which Publish Pose Estimate, Publish Pose, or Publish Point publish a point?

### DISCUSSION THREAD

**Discussion by swarajpeppermint** [foxglove/community#916] [Q&A] [answered]
![image](https://github.com/foxglove/community/assets/85288063/4b9134ba-47af-4aaf-bf5a-8d09d85f0a32)

How to change/namespace the topic to which these 3 options publish

**Comment by jtbandes** [comment, marked as answer]
Click the ⚙️ to access the panel settings. 

<img width="326" alt="image" src="https://github.com/foxglove/community/assets/14237/aa0f0cd7-e81a-4b1b-b2a2-7fdef2be645d">
